src/main/kotlin/com/nftco/flow/sdk/cadence/json-cadence.kt

```diff
@@ -197,8 +197,8 @@ open class Word8NumberField(value: String) : NumberField(TYPE_WORD8, value)
 open class Word16NumberField(value: String) : NumberField(TYPE_WORD16, value)
 open class Word32NumberField(value: String) : NumberField(TYPE_WORD32, value)
 open class Word64NumberField(value: String) : NumberField(TYPE_WORD64, value)
-open class Fix64NumberField(value: String) : NumberField(TYPE_FIX64, value)
-open class UFix64NumberField(value: String) : NumberField(TYPE_UFIX64, value)
+open class Fix64NumberField(value: String) : NumberField(TYPE_FIX64, value.addZeroDecimalIfNeeded())
```

Comment:
is it related to [json-cadence-marshalling.kt#L190-L191](https://github.com/portto/flow-jvm-sdk/blob/main/src/main/kotlin/com/nftco/flow/sdk/cadence/json-cadence-marshalling.kt#L190-L191)?